scripts/globals/status.lua

```diff
@@ -1557,9 +1558,9 @@ tpz.mod =
 
     -- The spares take care of finding the next ID to use so long as we don't forget to list IDs that have been freed up by refactoring.
     -- 570 - 825 used by WS DMG mods these are not spares.
-    -- SPARE = 958, -- stuff
-    -- SPARE = 959, -- stuff
-    -- SPARE = 960, -- stuff
+    -- SPARE = 964, -- stuff
+    -- SPARE = 965, -- stuff
+    -- SPARE = 966, -- stuff
```

Comment:
can you do me the favor of making sure modifier.h has the same "spares" as well?

 the fact you had to jump the 1st sppare id to 964 tells me the 2 fise have come unsynchornised in their lists again.

Reply:
not your fault you just happen to be in prime position to fix it

Reply:
https://github.com/neuromancerxi/topaz/blob/003425b91d553a271fd4237119def041b4cf3be0/src/map/modifier.h#L804

modifier.h already has next spare as 964.

Reply:
but I looked and it doesn't have 1:1 for all 3.

```cpp
    // The spares take care of finding the next ID to use so long as we don't forget to list IDs that have been freed up by refactoring.
    // 570 through 825 used by WS DMG mods these are not spares.
    // SPARE = 964, // stuff
    // SPARE = 965, // stuff
};
```
both files should have all matching id's, for everything, even the spares. it helps prevent future editors form mistakenly thinking an ID is used/not used